### PR TITLE
Bugfix/183897 fixing date picker

### PR DESCRIPTION
**Pull request by rodrigoxavier**
Fixing bug in the datepicker component.

### CODE REVIEW

File: src/components/blipDatepicker/index.js [hunk 1]
```diff
@@ -257,8 +255,7 @@ export class BlipDatepicker extends Component {
 
     const timeInputContainer = html`
     <div class="${BlipDatepicker.style.timeInputContainer}">
-      <div class="${BlipDatepicker.style.clockIcon}">${raw(Clock)}</div>
```
Comment (nicolasamaduro) [inline src/components/blipDatepicker/index.js:260]: Don't we have the clock Icon on the left side anymore?

File: src/components/blipDatepicker/index.js [hunk 2]
```diff
@@ -257,8 +255,7 @@ export class BlipDatepicker extends Component {
...
-      <input class="${BlipDatepicker.style.timeInput}" type="time" required>
+      <input class="${BlipDatepicker.style.timeInput}" type="date" required />
```
Comment (nicolasamaduro) [inline src/components/blipDatepicker/index.js:258]: This input expects a time format, does date type gives this format?

File: src/components/blipDatepicker/index.js (hunk 1 above)
Comment (rodrigoxavier) [reply src/components/blipDatepicker/index.js:260]: No. As talked to Anny, we decided to remove the first clock (blue) and keep only the right side clock that receive the click to open the time picker.

File: src/components/blipDatepicker/index.js (hunk 2 above)
Comment (rodrigoxavier) [reply src/components/blipDatepicker/index.js:258]: No. I'll fix it and create a new pull request.

Comment (nicolasamaduro) [reply src/components/blipDatepicker/index.js:258]: okkk

Comment (nicolasamaduro) [reply src/components/blipDatepicker/index.js]: You can commit in this same PR

**Pull request merged**